popup.js

```diff
@@ -0,0 +1,202 @@
+  tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+},
+{
+  id: 2,
+  name: 'Project name goes here...',
+  image: 'images/screen.png',
+  description: 'This is project that i have done in microverse ',
+  tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+},
+{
+  id: 3,
+  name: 'Project name goes here...',
+  image: 'images/Rectangle-21devices.png',
+  description: 'This is project that i have done in microverse ',
+  tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+},
+{
+  id: 4,
+  name: 'Project name goes here...',
+  image: 'images/Rectangle47_1.png',
+  description: 'This is project that i have done in microverse ',
+  tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+},
+{
+  id: 5,
+  name: 'Project name goes here...',
+  image: 'images/Rectangle-21device-3.png',
+  description: 'This is project that i have done in microverse ',
+  tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+},
+{
+  id: 6,
+  name: 'Project name goes here...',
+  image: 'images/Rectangle21device__4.png',
+  description: 'This is project that i have done in microverse ',
+  tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+},
+];
+
+const projectContainer = document.querySelector('.m-project-container');
+
+let projectTags = '';
+
+projectData.forEach((item) => {
+  projectTags += `
+    <img src=${item.image}  class="p-image"alt="work${item.id} image" id="work${item.id}img" />
+    <div class="project-name-${item.id} flex" id="project-name-${item.id}">
+      <h3 class="work-name">${item.name}</h3>
+      <div class="languages grid-container">
+        <div class="html language-title">${item.tech[0]}</div>
+        <div class="rails language-title">${item.tech[1]}</div>
+        <div class="js language-title">${item.tech[2]}</div>
+      </div>
+      <br>
+      <a class="see-this-project" id="work${item.id}" href="#">
+      <p class="text">${'See Project'}</p>
+      <img class="arrow " src="images/arrow.png " alt="icon-arrow ">
+  </a>
+ </div>`;
+});
+
+projectContainer.innerHTML = projectTags;
+
+const header = document.querySelector('header');
+const projects = document.querySelector('.container');
+const pagefooter = document.querySelector('.footer');
+const hamburger = document.querySelector('#modal-harmburger');
+
+let workopen = '#';
+
+function openModalWindow() {
+  const modalWindow = document.querySelector('#m-window');
+  const projId = this.id;
+  workopen += projId;
+
+  const projTitle = document.querySelector('#p-title');
+  projTitle.textContent = projectObj[projId].name;
+
+  const projImg = document.querySelector('#mw-img');
+  projImg.src = projectObj[projId].image;
+
+  const projTech = document.querySelectorAll('#l-items li');
+  projTech.forEach((item, i) => {
+    item.textContent = projectObj[projId].tech[i];
+  });
+
+  const projLive = document.querySelector('#btn-live');
+  projLive.href = projectObj[projId].live;
+
+  const projSource = document.querySelector('#btn-source');
+  projSource.href = projectObj[projId].source;
```

Comment:
- [ ] As pointed out in the first comment, populate the modal popup from the array of objects i.e `projectData`.